src/schemas/merokuDappStore.dAppStore.json

```diff
@@ -16,10 +16,14 @@
         "$ref": "merokuDappStore.featuredSchema.json"
       }
     },
-    "key": {
+    "storeKey": {
       "description": "Unique ID of this dApp Store",
       "type": "string"
     },
+    "key": {
```

Comment:
??

Reply:
this will be used for api call for a dapp,
storeId: kailash.appStore
key: kailash